app/decorators/ncr/work_order_decorator.rb

```diff
@@ -0,0 +1,28 @@
+module Ncr
+  class WorkOrderDecorator < Draper::Decorator
+    delegate_all
+
+    EMERGENCY_APPROVER_EMAIL = 'Emergency - Verbal Approval'
+    NO_APPROVER_FOUND        = 'No Approver Found'
```

Comment:
FWIW This is one of those things where I don't do `=` alignment, but also don't feel strongly enough that I would tell someone to change it.